src/components/cart/cart-view/cart-view.tsx

```diff
@@ -75,6 +79,13 @@ export const CartView = ({
                         >
                             Checkout
                         </button>
+                        <Link
```

Comment:
I know that @Letushev asked to use Link, but I don't agree with this change. It's a view component, and `onViewCart` should decide if it's navigation or maybe something else. On the practical matter - now to create a board for the card view it's required to wrap it in some Remix (or Router) context.

Reply:
In general I AGREE that links should be links, but one of the objectives of creating `CartView` component was to be able to create simple board without any mocks and context. The remix/react-router `Link` won't allow to do that.

Reply:
I will add Cart board in separate PR

Reply:
But we just need to wrap board component with `ComponentWrapper`. What's so bad about that?

Reply:
We can also keep `Link` and pass `cartPageUrl` prop.

Reply:
> What's so bad about that?
It shows that our view component is not "good enough" view :)

Reply:
> But we just need to wrap board component with ComponentWrapper. What's so bad about that?

`ComponentWrapper` wraps board component in `EcomApiProvider` and creates Remix stub. As a result board elements tree is not that straightforward and it's just adds unnecessary complexity for the user in my opinion.
So if choosing between simpler board and having this single link as a link I prefer simpler board.